### ISSUE TITLE
[core] Log "No files to analyze." message on INFO instead of warning

### ISSUE THREAD

**Issue by Bukama** [pmd/pmd#6236]
**Is your feature request related to a problem? Please describe.**
When a project contains no files to analyze, e.g. a Maven (sub-)project without Java files, PMD logs a warning about this:

> 16:06:50,986 [INFO] PMD version: 7.18.0
16:06:53,397 [WARNING] No files to analyze. Check input paths and exclude parameters, use --debug to see file collection traces.

CI-Pipelins who check for build warnings, collect these and report them as potential errors.
However there are several cases where a (sub-)project does not contains files to be analyzes by PMD, for example documentation or aggregating (sub-)projects. For those this warning can be seen as a false-positive.

**Describe the solution you'd like**
I suggest to either log this on INFO log level, like other SCA tools do (Spotbugs, Checkstyle, Jacoco) in general or provide an global config to do.

**Describe alternatives you've considered**
Only workaround is to add a full skip PMD execution config in each (sub-)project where a PMD execution is not necessary due missing files, which might be overseen if actual files are added to those and reduces the benefit of having global SCA configuration in a META-POM (~ company Parent-POM).

```xml
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-pmd-plugin</artifactId>
  <configuration>
     <skip>true</skip>
  </configuration>
</plugin>
```

**Additional context**
Low priority and ofc you can discuss if it is the right way to add skip-configuration in every affected (sub-)project and keep the warning in place. On the other site, I think if a report is always empty/not generated the user will check the log and find the INFO log message about missing files anyway.
